conda/common/io.py

```diff
@@ -533,6 +543,12 @@ def _tqdm(*args, **kwargs):
 
         return tqdm(*args, **kwargs)
 
+    @staticmethod
+    @lru_cache(maxsize=1)
+    def interactive():
+        """Return True if we're running in a terminal."""
+        return hasattr(sys.stdout, "isatty") and sys.stdout.isatty()
```

Comment:
I'm not a fan of static methods, could we just move this into a separate module level function or even just a constant?

```
is_interactive = hasattr(sys.stdout, "isatty") and sys.stdout.isatty()
```

Reply:
Done!